### ISSUE TITLE
Make this structure change (invert order of additional resources and FAQs)

### ISSUE THREAD

**Issue by BenGitsCode** [ga-wdi-boston/sql-study#90]
```diff
## Instructions

1.  Fork and clone this repository.
1.  Change into the new directory.
1.  Create and checkout a new branch, named `response`.
1.  Follow the directions given in [`study.md`](study.md).
1.  When finished, push to your fork and submit a pull request.


+ You may wish to refer to [FAQs](https://github.com/ga-wdi-boston/meta/wiki/)
+ related to [forking,
+ cloning](https://github.com/ga-wdi-boston/meta/wiki/ForkAndClone), and [pull
+ requests](https://github.com/ga-wdi-boston/meta/wiki/PullRequest).


## Additional Resources (optional)

 [Head First Labs Sql Hands On](http://www.headfirstlabs.com/sql_hands_on/)

- You may wish to refer to [FAQs](https://github.com/ga-wdi-boston/meta/wiki/)
- related to [forking,
- cloning](https://github.com/ga-wdi-boston/meta/wiki/ForkAndClone), and [pull
- requests](https://github.com/ga-wdi-boston/meta/wiki/PullRequest).

## [License](LICENSE)

1.  All content is licensed under a CC­BY­NC­SA 4.0 license.
1.  All software code is licensed under GNU GPLv3. For commercial use or
    alternative licensing, please contact legal@ga.co.

```

**Comment by jordanallain** [comment]
This appears to have been addressed but maybe not linked to in the commit message. Should we close here with the button or reference in a commit that doesn't actually address this?

**Comment by jordanallain** [comment]
@BenGitsCode shall we close this issue? looks to be addressed in ddaa5ea

**Comment by BenGitsCode** [comment]
Okay to close. 👍
